lib/galaxy/datatypes/binary.py

```diff
@@ -1107,6 +1107,63 @@ def display_peek(self, dataset):
             return "Cool (HDF5) file (%s)." % (nice_size(dataset.get_size()))
 
 
+class MCool(Cool):
+    """
+    Class describing the multi-resolution cool format (https://github.com/mirnylab/cooler)
+    """
+
+    file_ext = "mcool"
+
+    def sniff(self, filename):
+        """
+        >>> from galaxy.datatypes.sniff import get_test_fname
+        >>> fname = get_test_fname('matrix.mcool')
+        >>> MCool().sniff(fname)
+        True
+        >>> fname = get_test_fname('test.mz5')
+        >>> MCool().sniff(fname)
+        False
+        >>> fname = get_test_fname('wiggle.wig')
+        >>> MCool().sniff(fname)
+        False
+        >>> fname = get_test_fname('biom2_sparse_otu_table_hdf5.biom2')
```

Comment:
can we add here a cool test what will eval to False?

Reply:
Yes. Done